targets/cloudscript-typings/make.js

```diff
@@ -26,3 +28,122 @@ exports.makeServerAPI = function (apis, sourceDir, apiOutputDir) {
     writeFile(path.resolve(outputDir, "CloudScript.d.ts"), generatedApi);
 
 }
+
+/**
+  Handles selecting the correct datatype template to use for a datatype.
+  enum, interface
+*/
+function MakeDatatype(datatype, api, sourceDir) {
+  var stringLiteralTemplate = compileTemplate(sourceDir,"StringLiteral");
+  var interfaceTemplate = compileTemplate(sourceDir,"Interface");
```

Comment:
Calls to EJS are reasonably expensive (It loads and compiles an entire file)
I would not fetch both templates here but rather...